inc/customizer/controls/react/src/icon-color/IconColorComponent.js

```diff
@@ -0,0 +1,12 @@
+import IconColor from './IconColor';
```

Comment:
IconColor is not a control, but part of the Repeater control. This means it doesn't need a file that interacts with the customizer API. 👯